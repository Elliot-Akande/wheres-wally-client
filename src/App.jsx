function App() {
  return (
    <>
      <h1>Where&apos;s Wally?</h1>
    </>
  );
}

export default App;
