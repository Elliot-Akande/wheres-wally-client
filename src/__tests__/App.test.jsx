import { render, screen } from "@testing-library/react";

import App from "../App";

describe("App", () => {
  it("renders headline", () => {
    const { container } = render(<App title="React" />);

    screen.debug();

    expect(container).toMatchSnapshot();
  });
});
