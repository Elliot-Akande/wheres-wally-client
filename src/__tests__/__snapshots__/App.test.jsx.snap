// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > renders headline 1`] = `
<div>
  <h1>
    Where's Wally
  </h1>
</div>
`;
